packages/okta-react/src/AuthService.js

```diff
@@ -256,17 +257,14 @@ class AuthService {
   }
 
   on( event, callback ) {
-    const subscriberId = this._subscriberCount++;
-    this._listeners[event] = this._listeners[event] || {};
-    this._listeners[event][subscriberId] = callback;
-    return () => { 
-      delete this._listeners[event][subscriberId];
-    }
+    this._oktaAuth.emitter.on(event, callback, this._oktaAuth.emitter);
```

Comment:
This looks like it lets consumers subscribe to the auth-js events, which is a different thing than the exist okta-react authStateChange event.

We want to push those events to auth-js, but auth-js doesn't yet have the concept of authStateChange, so I'm not clear on how this works.   (Note Security.js uses the authStateChange event)

Is there something I'm missing?

Reply:
Auth-js has similar event emitter as implemented in the react SDK. The change here is just to use the `emitter` exposed by auth-js without introducing a breaking change.

Reply:
So this still emits the authStateChange event, just using the upstream emitter?   That feels...weird.

You removed the test that checks for the authStateChange event - can we add in a version that checks to see that it can be subscribed to?